sig-ui/annual-report-2021.md

```diff
@@ -4,15 +4,14 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
```

Comment:
One thing that I think may be worth mentioning here is that we have significantly updated our config, scripts etc. This opens space for multiple improvements that were blocked before.